compiler/il/OMRResolvedMethodSymbol.cpp

```diff
@@ -2387,6 +2388,32 @@ OMR::ResolvedMethodSymbol::setUsesSinglePrecisionMode(bool b)
    _methodFlags.set(SinglePrecisionMode, b);
    }
 
+bool
+OMR::ResolvedMethodSymbol::isParmVariant(TR::ParameterSymbol * parmSymbol)
+   {
+   if (_variantParms == NULL)
+      self()->detectVariantParms();
+   TR_ASSERT_FATAL(parmSymbol->getOrdinal() < self()->getNumParameterSlots(),
```

Comment:
Number of parameters may not match number of parameter slots. I think we should compare parm ordinal to number of parameters, or compare parm slot to numberof parameter slots.

Reply:
For reference, there is `numberOfParameters` on resolved method if you want to use.

Reply:
Fixed in force push.